ksqldb-rest-app/src/test/java/io/confluent/ksql/rest/server/execution/ListConnectorsExecutorTest.java

```diff
@@ -212,4 +226,94 @@ public void shouldListInvalidConnectorWithNoInfo() {
     )));
   }
 
+  @Test
+  public void shouldReturnPluggableForbiddenError() {
+    //Given:
+    when(connectClient.connectors())
+        .thenReturn(
+            ConnectResponse.failure("FORBIDDEN", HttpStatus.SC_FORBIDDEN));
+
+    final KsqlConfig config = new KsqlConfig(ImmutableMap.of(
```

Comment:
nit: we can reduce code repetition and improve readability of unit tests by pulling out common setup code like this into a helper method such as `givenCustomErrorHander();`. No need to go through and update all your newly added tests if it's too much toil but definitely good to keep in mind for the future.

Reply:
Good idea! Done in dcc2852ac34031b32089af45f6b8ebfa71e1edd8.